_posts/en/newsletters/2019-12-18-newsletter.md

```diff
@@ -0,0 +1,179 @@
+---
+title: 'Bitcoin Optech Newsletter #77'
+permalink: /en/newsletters/2019/12/18/
+name: 2019-12-18-newsletter
+slug: 2019-12-18-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter announces the release of LND 0.8.2-beta, requests
+help testing the latest C-Lightning release candidate, discusses
+widespread support for basic multipath payments in LN, provides an update on
+bech32 error detection reliability, summarizes updates to the proposed
+`OP_CHECKTEMPLATEVERIFY` opcode, and links to a discussion about the impact
+of eclipse attacks on LN channels.  Also included is our regular section
+about notable changes to popular Bitcoin infrastructure projects. <!--
+FIXME: also mention other sections if schmidty decides to write them -->
+
+{% comment %}<!-- include references.md below the fold but above any Jekyll/Liquid variables-->{% endcomment %}
+{% include references.md %}
+
+## Action items
+
+- **Upgrade to LND 0.8.2-beta:** this [release][lnd 0.8.2-beta] contains
+  several bug fixes and minor UX improvements, most notably for the
+  recovery of static channel backups.
+
+- **Help test C-Lightning 0.8.0 RCs:** the [release candidate][cl 0.8.0]
+  for the next version of C-Lightning switches the default network to
+  mainnet instead of testnet (see [Newsletter #75][news75 cl mainnet])
+  and adds support for basic multipath payments (described below) among
+  many other features and bug fixes.
+
+## News
+
+- **LN implementations add multipath payment support:** after a
+  significant amount of discussion and development, all three LN
+  implementations tracked by Optech have now added basic support for
+  [multipath payments][topic multipath payments].  A multipath payment
+  consists of multiple LN payments routed over different paths which may
+  all be claimed at the same time by the receiver.  This provides a
+  major upgrade to the usability of LN by allowing users with their
+  balance split across multiple channels to spend or receive funds in
+  multiple channels in the same overall payment.  With this upgrade,
+  normal non-routing users don't need to worry about how much balance or
```

Comment:
Changing it to only apply the strong benefit to spenders, which I think is correct since they know the balances of all their channels and so can split the payment accurately on the first try.

I think there's a pretty significant benefit for receivers too since each individual part of a multipath payment can be resent independetly of the other parts, so an implementation that automatically resends failed parts may be able to quickly find a working set of path/amounts without the whole payment failing.  (The current spec allows a minimum of 60 seconds between when the first part of a multipath payment arrives and when the last part is required to arrive, so there's plenty of time for retying; though maybe that tolerance will be reduced in later spec updates.)  However, that's getting into the weeds a bit, so I'm not describing this in the newsletter; I'll just leave it as describing the benefit for spenders.

Reply:
Agree with all of that!